packages/brisa/src/utils/handle-css-files/index.ts

```diff
@@ -2,126 +2,118 @@ import path from 'node:path';
 import type { Loader } from 'bun';
 import fs from 'node:fs';
 import { getConstants } from '@/constants';
-import { logError } from '../log/log-build';
```

Comment:
Why clear the logError with the try-catch? This is to give error feedback of the build in the terminal.

Reply:
Ok, it's fine because the error is handled upper